LoopKit/DeviceManager/CGMManager.swift

```diff
@@ -111,6 +111,11 @@ public protocol CGMManager: DeviceManager {
     ///
     /// - Parameter observer: The observing object
     func removeStatusObserver(_ observer: CGMManagerStatusObserver)
+
+    /// Requests that the manager completes its deletion process
+    ///
+    /// - Parameter completion: Action to take after the CGM manager is deleted
+    func deleteCGMManager(completion: @escaping () -> Void)
```

Comment:
[nit] seems a bit odd that a function called `deleteCGMManager` is on `CGMManager`.  Maybe call it `completeDeletion` or just `delete`?

Reply:
@rickpasetto I was going to name it `delete`, but then I thought this might be too general (delete what?).

@darinkrauss if you also agree with `delete`, please let me know.

Reply:
@rickpasetto just realized it would be `cgmManager.delete()` which is clear enough. I'll update.